### ISSUE TITLE
PAC not supported in Java 15

### ISSUE THREAD

**Issue by StFS** [akuhtz/proxy-vole#19]
Java 15 has removed Nashorn (https://openjdk.java.net/jeps/372).

However, there is a standalone Nashorn project (https://mail.openjdk.java.net/pipermail/nashorn-dev/2020-October/007557.html) now. It should be possible to declare a dependency on that project to get everything working with Java 15.

Unfortunately, I don't have time right now to look into this and find a fix, but if I find the time I will create a PR.

**Comment by StFS** [comment]
I also noticed that the unit tests do not run on Java 11+.

That issue is trivial as it seems that's just a matter of the `toString()` method of `InetSocketAddress` having changed. It appends `/unresolved` to the host name when an instance is created using the `InetSocketAddress.createUnresolved(String, int))` method.

**Comment by akuhtz** [comment]
@StFS I just uploaded 1.0.12-SNAPSHOT to the sonatype repo. The tests now pass on Java 11+.

Can you test with this version on Java 15? I switched to rhino some time ago.

**Comment by akuhtz** [comment]
I think we should wait for this: https://github.com/javadelight/delight-nashorn-sandbox/issues/109

**Comment by StFS** [comment]
I'm so sorry. I didn't even test this on your version of the module, just assumed that this fork would have the same issue since the unit tests were failing in the same way.

I've tested my application with your module (version 1.0.11) and that works fine. I don't even need the snapshot version for the PAC mechanism to work since you are using a different JS engine.

I also tried running the unit tests with Java 15 and that works fine on the develop branch.

So I'm pretty sure I'm a happy camper here. I'm closing this issue. Thanks!
